snakemake_executor_plugin_slurm/utils.py

```diff
@@ -1,6 +1,9 @@
 # utility functions for the SLURM executor plugin
 
 import os
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
```

Not needed  I think. Snakemake has its own logger instance that we access with `self.logger`.